source/development_process/2_deployment.rst

```diff
@@ -50,16 +50,16 @@ string.
 The Architecture of Chef, Abridged
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Chef has several parts, a central server which holds all of the cookbooks, and
+Chef has several parts: a central server, which holds all of the cookbooks, and
 a collection of nodes, which are managed by the chef server. Each node has
 attributes which are essentially a hash or a dictionary of configuration values
 and keys. Each node has a list of roles it serves, like web server or database
-server, and a run list of recipes it will run. Every half hour or so each node
+server, and a run list of recipes it will run. Every half hour or so, each node
```

Comment:
The first sentence that ends on this line could be refactored. Run lists actually contain both recipes and roles. Perhaps

```
Each node has a run list containing recipes and/or roles. A role is a collection of recipes and attributes.
```